lib/active_admin/helpers/collection.rb

```diff
@@ -15,7 +15,7 @@ def collection_size(collection=collection)
         size
       end
 
-      def collection_is_empty?(collection=collection)
+      def collection_is_empty?(collection=collection())
```

Comment:
Do not use parentheses for method calls with no arguments.<br>Surrounding space missing in default value assignment.